src/main/scala/cromwell/engine/backend/local/SharedFileSystem.scala

```diff
@@ -28,6 +31,7 @@ object SharedFileSystem {
     case "copy" => localizePathViaCopy _
     case unsupported => throw new UnsupportedOperationException(s"Localization strategy $unsupported is not recognized")
   })
+  val CallLocalizers = Stream(localizePathAlreadyLocalized _, localizePathViaSymbolicLink _, localizePathViaHardLink _, localizePathViaCopy _)
```

Comment:
I see your point but I think we just disagree on this :smile: (@scottfrazer get out of this body !)
I see this call re-localization thing as an internal cromwell process which doesn't have/need to align with user first-pass localization strategies defined in the conf.
Even if the user decides to localize all its inputs with "copy" strategy, which will be done when localizing to workflow-inputs, I think we could still try to use links to re-localize to calls "inputs" directories.